src/component/feature/FeatureView/FeatureVariants/FeatureVariantsList/AddFeatureVariant/AddFeatureVariant.tsx

```diff
@@ -388,20 +383,26 @@ const AddVariant = ({
                         <ConditionallyRender
                             condition={payload.type === 'json'}
                             show={
-                                <CodeEditor
-                                    value={code}
-                                    language="json"
-                                    placeholder="Please enter JSON payload."
-                                    onChange={e => setCode(e.target.value)}
-                                    padding={15}
-                                    style={{
-                                        fontSize: 12,
-                                        backgroundColor: '#f5f5f5',
-                                        fontFamily:
-                                            'ui-monospace,SFMono-Regular,SF Mono,Consolas,Liberation Mono,Menlo,monospace',
-                                        minHeight: 5,
-                                    }}
-                                />
+                                <>
+                                    <p style={{ color: 'red' }}>
```

Comment:
Maybe we should move the error below the field, so that the field doesn't move when there's an error.

Should also add some a11y attributes here, to both the editor and the error.